### PR TITLE
AO3-4809 - Extend test coverage in Bookmarks Controller

### PR DESCRIPTION
**Pull request by scottsds**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4809

## Purpose

Extends tests for the Bookmarks Controller.

## Testing

Use Coveralls to confirm that the coverage percentage has increased:
1. Go to https://coveralls.io/github/otwcode/otwarchive?branch=master
2. In the "Search" field near the "Source Files on Master" heading, enter the name of the controller: bookmarks_controller
3. Note the number in the "Coverage" column of the table, it should be greater than 79.88%

### CODE REVIEW

**Comment by sarken** [comment]
Marking needs coder action due to chat discussion

**Comment by scottsds** [comment]
I have restored the code I deleted (whoopsies), this should be ready for a review now.

File: features/step_definitions/admin_steps.rb [hunk 1]
```diff
@@ -70,6 +70,20 @@ def update_settings(settings)
   Language.find_or_create_by_short_and_name("FR", "Francais")
 end
 
+Given /^caching is enabled$/ do
```
Comment (sarken) [inline features/step_definitions/admin_steps.rb]: I think these changes might have snuck in from somewhere else? It doesn't look like they're being used in the cukes.

Comment (zz9pzza) [inline features/step_definitions/admin_steps.rb]: It looks that way.

Comment (scottsds) [inline features/step_definitions/admin_steps.rb]: I was going to use them to further test the Controller but couldn't figure it out, but thought these step defs would be good to keep for future tests. I can pull them out if you'd like. Let me know.

Comment (sarken) [inline features/step_definitions/admin_steps.rb]: Maybe take them out for now? We've kind of been mumbling about perhaps getting rid of these settings.

**Comment by scottsds** [comment]
Removed the step defs that weren't used.

**Comment by scottsds** [comment]
@sarken I have updated the PR to reflect the changes you requested regarding taking the logic out of the flash message. I have also made sure that the flash message is being automatically checked by a cuke. I left notes on the issue so that testers can manually test as well.

**Comment by sarken** [comment]
Looks good to me!

**Pull request merged**